src/customer/structures.ts

```diff
@@ -150,7 +153,7 @@ export interface LoginRequest {
   customer_side_storage?: object;
   group_id?: number;
   referrer?: string;
-  pushes?: { [key: string]: Pushes[]};
+  pushes?: { [key: string]: Pushes[] };
```

Comment:
this definition is inconsistent with Agent part ;)

also, did you decide to ditch the idea of push - version mapping?

Reply:
Yes, I think versioning pushes by Typescript's types will be too overhead without any gain (excluding autocomplete in some editors) and it'll increase the cost of maintaining the code.